src/ripple/app/tx/impl/SetAccount.cpp

```diff
@@ -201,38 +208,37 @@ SetAccount::preclaim(PreclaimContext const& ctx)
 TER
 SetAccount::doApply ()
 {
-    std::uint32_t const uTxFlags = ctx_.tx.getFlags ();
-
-    auto const sle = view().peek(
-        keylet::account(account_));
+    auto const sle = view().peek(keylet::account(account_));
 
     std::uint32_t const uFlagsIn = sle->getFieldU32 (sfFlags);
     std::uint32_t uFlagsOut = uFlagsIn;
 
-    std::uint32_t const uSetFlag = ctx_.tx.getFieldU32 (sfSetFlag);
-    std::uint32_t const uClearFlag = ctx_.tx.getFieldU32 (sfClearFlag);
+    STTx const& tx {ctx_.tx};
+    std::uint32_t const uSetFlag {tx.getFieldU32 (sfSetFlag)};
+    std::uint32_t const uClearFlag {tx.getFieldU32 (sfClearFlag)};
 
     // legacy AccountSet flags
-    bool bSetRequireDest   = (uTxFlags & TxFlag::requireDestTag) || (uSetFlag == asfRequireDest);
-    bool bClearRequireDest = (uTxFlags & tfOptionalDestTag) || (uClearFlag == asfRequireDest);
-    bool bSetRequireAuth   = (uTxFlags & tfRequireAuth) || (uSetFlag == asfRequireAuth);
-    bool bClearRequireAuth = (uTxFlags & tfOptionalAuth) || (uClearFlag == asfRequireAuth);
-    bool bSetDisallowXRP   = (uTxFlags & tfDisallowXRP) || (uSetFlag == asfDisallowXRP);
-    bool bClearDisallowXRP = (uTxFlags & tfAllowXRP) || (uClearFlag == asfDisallowXRP);
-
-    bool sigWithMaster = false;
+    std::uint32_t const uTxFlags {tx.getFlags ()};
+    bool const bSetRequireDest   {(uTxFlags & TxFlag::requireDestTag) || (uSetFlag == asfRequireDest)};
+    bool const bClearRequireDest {(uTxFlags & tfOptionalDestTag) || (uClearFlag == asfRequireDest)};
+    bool const bSetRequireAuth   {(uTxFlags & tfRequireAuth) || (uSetFlag == asfRequireAuth)};
+    bool const bClearRequireAuth {(uTxFlags & tfOptionalAuth) || (uClearFlag == asfRequireAuth)};
+    bool const bSetDisallowXRP   {(uTxFlags & tfDisallowXRP) || (uSetFlag == asfDisallowXRP)};
+    bool const bClearDisallowXRP {(uTxFlags & tfAllowXRP) || (uClearFlag == asfDisallowXRP)};
 
+    bool const sigWithMaster {[tx, acct = account_] ()
```

Comment:
You are copying `tx` and `acct` here, I think you want refs.

Note: for an immediately invoked function expression, such as this one, I prefer capture all by ref `[&]`. What the lambda actually captures isn't important in such a context.